sdk-lib/src/androidTest/java/cash/z/ecc/android/sdk/integration/SanityTest.kt

```diff
@@ -42,13 +42,13 @@ class SanityTest(
     fun testFilePaths() {
         assertEquals(
             "$name has invalid DataDB file",
-            "/data/user/0/cash.z.ecc.android.sdk.test/databases/TestWallet_${networkName}_Data.db",
-            wallet.initializer.rustBackend.pathDataDb
+            "/data/user/0/cash.z.ecc.android.sdk.test/no_backup/co.electricoin.zcash/TestWallet_${networkName}_Data.db",
```

Comment:
`/data/user/0/` is something that might potentially break in the future e.g. if the tests are run on a device with multiple users

Do you think `wallet.initializer.rustBackend.dataDbFile.absolutePath.endsWith("no_backup/co.electricoin.zcash/TestWallet_${networkName}_Data.db")` might be more robust?

Reply:
Yea, definitely, fixed that too. I also searched for another occurrences of similar in tests.